tools/cluster_mgr.py

```diff
@@ -271,30 +308,32 @@ def pack(slot_ranges):
 
 def migrate(args):
     config = build_config_from_existing(args)
-    target = find_node(config, args.target_port)
-    target_node = Node(target["master"]["port"])
+    target = find_node(config, args.target_host, args.target_port)
+    target_node = Node(target["master"]["ip"], target["master"]["port"])
```

Comment:
Maybe insert update_id in Node construction?

Reply:
That's unfortunately not possible because in `create_locally` we first create the node, and then start it. I could refactor some code I guess but I don't think it's worth it.
But I did move it to be a method instead of a free standing function :)